src/lib.rs

```diff
@@ -91,3 +96,12 @@ pub unsafe fn JS_ARGV(_cx: *mut JSContext, vp: *mut JSVal) -> *mut JSVal {
 pub unsafe fn JS_CALLEE(_cx: *mut JSContext, vp: *mut JSVal) -> JSVal {
     *vp
 }
+
+// This is measured properly by the heap measurement implemented in SpiderMonkey.
+impl<T: Copy + GCMethods<T>> HeapSizeOf for Heap<T> {
+    fn heap_size_of_children(&self) -> usize {
+        0
+    }
+}
+known_heap_size!(0, JSVal);
```

Comment:
That's https://github.com/michaelwu/rust-bindgen/ which also requires custom patches to clang among other things. We currently hand-edit these things. Anyways, I don't think it's a big deal at this point and we can move on.